concourse/pipelines/container-build.jsonnet

```diff
@@ -194,9 +203,61 @@ local BuildContainerImage(image) = buildcontainerimgjob {
             },
           }
           for arch in daisy_architectures
-        ] +
-        //  Put three binaries using gsutil.
-        [
+        ],
+      plan:
+        super.plan + [
+          {
+            task: 'run-daisy-integ-tests',
+            config: {
+              inputs: [{ name: 'compute-daisy' }],
+              params: {
+                // Force the test runner to use application default credentials,
+                // which are available through the k8s metadata server.
+                GOOGLE_APPLICATION_CREDENTIALS: '',
+              },
+              platform: 'linux',
+              image_resource: {
+                type: 'registry-image-forked',
+                source: {
+                  repository: 'gcr.io/compute-image-tools-test/test-runner',
+                  tag: 'latest',
+                  google_auth: true,
+                },
+              },
+              run: {
+                path: '/daisy_test_runner',
+                args: [
+                  '-projects=compute-image-test-pool-001',
+                  '-zone=us-central1-c',
+                  'compute-daisy/daisy_integration_tests/daisy_e2e.test.gotmpl',
```

Comment:
it's not 100% clear to me how this tests the newly built daisy without any reference to it; looking at it, it builds daisy test runner from source, and has no relation to the daisy image build. should we link them together somehow?

Reply:
Oh interesting -- since the daisy test runner is built at [L138](https://github.com/GoogleCloudPlatform/guest-test-infra/blob/5c7e8aec92201eee01a68f433ecff5c28b61d96c/concourse/pipelines/container-build.jsonnet#L138), my assumption was this would reference *that* build result. Is my assumption incorrect? If so, what would you suggest?

Reply:
These are currently independent jobs with no relationship, so presumably they could be run in any order or even simultaneously. To fix, if that test runner is also updated to tag with the git commit like we're doing here, we could then add a `passed:[]` type dependency to that job, and reference the image by tag rather than by 'latest' to ensure we're testing the same code. In fact why don't we update the base template to add the step to get the commit and also tag it, so this becomes a norm for all these images?

Reply:
good idea! done.